docs/howto/autodetect-fractions/pom.xml

```diff
@@ -89,7 +89,6 @@
       </plugin>
       <plugin>
         <artifactId>maven-failsafe-plugin</artifactId>
-        <version>${version.maven-failsafe-plugin}</version>
```

Comment:
We can use the version coming in from WF Parent directly .. or if need be, we can manage it from root POM.